src/main/java/wooteco/subway/ui/StationController.java

```diff
@@ -1,39 +1,46 @@
 package wooteco.subway.ui;
 
+import java.net.URI;
+import java.util.List;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
-import wooteco.subway.dao.StationDao;
-import wooteco.subway.domain.Station;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
 import wooteco.subway.dto.StationRequest;
 import wooteco.subway.dto.StationResponse;
-
-import java.net.URI;
-import java.util.List;
-import java.util.stream.Collectors;
+import wooteco.subway.service.StationService;
 
 @RestController
 public class StationController {
 
+    private final StationService stationService;
+
+    public StationController(StationService stationService) {
+        this.stationService = stationService;
+    }
+
     @PostMapping("/stations")
     public ResponseEntity<StationResponse> createStation(@RequestBody StationRequest stationRequest) {
-        Station station = new Station(stationRequest.getName());
-        Station newStation = StationDao.save(station);
-        StationResponse stationResponse = new StationResponse(newStation.getId(), newStation.getName());
-        return ResponseEntity.created(URI.create("/stations/" + newStation.getId())).body(stationResponse);
+        final StationResponse response = stationService.save(stationRequest);
+
+        return ResponseEntity.created(URI.create("/stations/" + response.getId())).body(response);
     }
 
     @GetMapping(value = "/stations", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<List<StationResponse>> showStations() {
-        List<Station> stations = StationDao.findAll();
-        List<StationResponse> stationResponses = stations.stream()
-                .map(it -> new StationResponse(it.getId(), it.getName()))
-                .collect(Collectors.toList());
-        return ResponseEntity.ok().body(stationResponses);
+        final List<StationResponse> response = stationService.findAll();
+
+        return ResponseEntity.ok().body(response);
```

Comment:
```suggestion
        return ResponseEntity.ok(response);
```

사소한 것이긴 하지만, OK 응답의 경우 이렇게 shortcut을 지원하고 있어요!

Reply:
`A shortcut for creating a ResponseEntity with the given body and the status set to OK.`와 같이
body()를 생략하는 shortcut을 제공하는지 몰랐네요..!! 감사합니다!!😁